h2o-genmodel/src/main/java/hex/genmodel/AbstractMojoWriter.java

```diff
@@ -247,6 +248,22 @@ private void writeDomains() throws IOException {
       finishWritingTextFile();
     }
   }
+  
+  /**
+   * Creates section `target_encoding_map` in model.ini file that store keys for retrieving map for target encoding
+   */
+  private void writeTargetEncodingMap() throws IOException {
+    startWritingTextFile("feature_engineering/targetencoding.ini");
```

Comment:
I guess this is still WIP, but I'd go with something like features/target_encoding.whatever

Reply:
@Pscheidl Yes, it is kind of a WIP. I wanted to get highlevel review to understand if I'm doing right things at all in terms of design/location of the logic e.t.c.
Regarding suggestion I would not go with 'features' as it is too generic even given the fact that we are doing data science here. Underscore for the second part is a good idea. So, 'feature_engineering/target_encoding.ini' ?